nmi_mysql/nmi_mysql.py

```diff
@@ -11,6 +11,7 @@
 
 CONFIG_KEYS = ['host', 'user', 'password', 'db', 'port']
 MAX_POOL_SIZE = 10
+MAX_RETRIES = 5
```

Comment:
Ah, I got confused because MAX_POOL_SIZE and MAX_RETRIES are actually used differently. 
Let's make them do similar things?

Also, we shouldn't care if the user wants to set to 100 retries unless it breaks our library.